Libraries/Text/TextInput/Multiline/RCTUITextView.m

```diff
@@ -145,26 +145,6 @@ - (void)setTextAlignment:(NSTextAlignment)textAlignment
   _placeholderView.textAlignment = textAlignment;
 }
 
-- (void)setAttributedText:(NSAttributedString *)attributedText
```

Comment:
Could you explain why this is redundant and safe to remove?

Reply:
Root cause of this issue is updating `attributedText` via `-setAttributedText:` of `RCTBaseTextInputView`.
In this removed block, it attempts to avoid calling the method by comparing two attributed text with bare strings. But it should be handled by `-textOf:quals:` which already handles some other special cases such as Chinese and Japanese. 
I added detection for Korean to safely fallbacks to bare text comparison in the `-textOf:quals:`.

Reply:
I found out that it is required to invalidate placeholder visibility in this method. I added a required commit.